badge/badge.css

```diff
@@ -0,0 +1,128 @@
+@import "../resetcss.css";
+
+
+.skin{
+    padding: 0.5rem 0.5rem 0 0.1rem;
+    
+}
+.card-skin{
+    margin: 1rem;
+    width: 35rem;
+}
+
+a{
+  font-size: 2rem;
+  text-decoration: none;
+  color: black;
+}
+
+  article a{
+      display: block;
+      margin-top: 1rem;
+      padding-left: 2rem;
+      
+  
+  }
+  article{
+      display: flex;
+      flex-direction: column;
+      gap: 1rem;
+      /* margin : 2rem 0 8rem 0; */
+  }
+  
+
+    /*-------------------------------------------------avatar  */
+
+    .badge-box{
+        border: 1px solid var(--primary-color);
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        width: 60%;
+        margin : 2rem auto 8rem auto;
+        
+    }
+    .btn-notification{
+        border: none;
+        margin: 2rem 0;
+        padding: 1rem;
+        background: transparent;
+        position: relative;
+    }
+    .notification-number{
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        position: absolute;
+        border: 1px solid black;
+        /* padding: 1rem; */
```

Comment:
you can remove this commented code and there are few places where you commented some code